cmake/BundledZLIB.cmake

```diff
@@ -50,7 +50,7 @@ function(use_bundled_zlib SOURCE_DIR BINARY_DIR)
     set(ZLIB_FOUND "YES" CACHE STRING "" FORCE)
     set(ZLIB_INCLUDE_DIRS "${BINARY_DIR}/thirdparty/zlib-install/include" CACHE STRING "" FORCE)
     set(ZLIB_LIBRARIES "${BINARY_DIR}/thirdparty/zlib-install/${BYPRODUCT}" CACHE STRING "" FORCE)
-    set(ZLIB_VERSION_STRING "1.2.11" CACHE STRING "" FORCE)
+    set(ZLIB_VERSION_STRING "1.3.1" CACHE STRING "" FORCE)
     set(ZLIB_VERSION_MAJOR 1 CACHE STRING "" FORCE)
     set(ZLIB_VERSION_MINOR 2 CACHE STRING "" FORCE)
     set(ZLIB_VERSION_PATCH 11 CACHE STRING "" FORCE)
```

Comment:
I'm not sure what these are needed for, but we should update them, as well, just to be safe. Or if you're sure they're not needed, we can remove them.